src/components/Footer/subscribeSection.js

```diff
@@ -39,89 +40,111 @@ const SubscribeSection = ({ size }) => {
     case 'duplicate':
       return (
         <Typography variant='body2' className={classes.errorMessage}>
-          That email address has already been registered with us.
+            That email address has already been registered with us.
```

Comment:
Why is there extra indentation here?

Reply:
I did format document and due to which it added an extra space. I will change it